src/main.js

```diff
@@ -3,6 +3,7 @@ export { default as ActionBar } from './components/ActionBar';
 export { default as ActionItem } from './components/ActionItem';
 export { default as NavigationButton } from './components/NavigationButton';
 export { default as Label } from './components/Label';
+export { default as Page } from './components/Page';
```

Comment:
this should be removed and Page should be imported where it is needed.

Reply:
Page has been removed from main.js